docs/metrics.md

```diff
@@ -0,0 +1,161 @@
+## Thanos General Metrics
+
+### List of General Metrics Exported By Thanos
```

Comment:
The two headers seem to mean the exact same thing. Also the other headers don't seem well named, imho. Below you can find my suggestion, but feel free to use anything that you might find better.

```suggestion
## General Metrics

Some metrics are exported by certain internal subsystems that can be used by one or more Thanos components. Below you can find a list of these metrics.

### Thanos metrics

(...)

### Store API metrics

(...)

###
```

Reply:
I will align both headers accordingly.